docs/start/getting-started/fragments/android/integrate.md

```diff
@@ -3,6 +3,7 @@ Next you'll use the generated model to read and write data. In this section you'
 ## Configure Amplify and DataStore
 
 First, we'll add the DataStore plugin and configure Amplify by creating an Application class and overriding the `onCreate()` method.
+1. Navigate to the **example.todo** folder located at **ToDo** > **app** > **src** > **main** > **java/kotlin** > **com** > **example.todo**
```

Comment:
The `.` in `example.todo`:

This _probably is_ a valid instruction, if the user is navigating a UI pane in Android Studio. Android Studio will break from listing _directory structure_, and instead list _Java packages_ in some cases. In Android projects, directory structure and java package identifiers match, except for `/` being written as `.`, instead.

Not sure if this is worth clarifying, or how best to do it (without getting into trivialities.)

TLDR: `example.todo` is actually a directory path itself, `**example** > **todo**`.